jobs/JGLOBAL_SNOW_ANALYSIS

```diff
@@ -18,12 +17,15 @@ GDUMP="gdas"
 # Begin JOB SPECIFIC work
 ##############################################
 # Generate COM variables from templates
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_OBS COM_SNOW_ANALYSIS COM_CONF
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+    COMIN_OBS:COM_OBS_TMPL \
+    COMOUT_SNOW_ANALYSIS:COM_SNOW_ANALYSIS_TMPL \
+    COMOUT_CONF:COM_CONF_TMPL
```

Comment:
Is this still needed if yamls aren't being copied here? (See above comment)

Reply:
You mean the COMOUT_CONF? probably not, I can remove that